### PR TITLE
Adding EKS input type to the cloud security posture integration

### PR DESCRIPTION
**Pull request by ofiriro3**
## What does this PR do?
This PR is enhancing the current Cloud security posture integration by adding another input type that supports collecting findings against an EKS cluster.
The new input type defines new input fields AKA the AWS credentials that should be sent as part of creating the integration.
 

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues
- Closes https://github.com/elastic/security-team/issues/2991
- https://github.com/elastic/elastic-agent/pull/638

### CODE REVIEW

File: packages/cloud_security_posture/data_stream/findings/manifest.yml [hunk 1]
```diff
@@ -1,6 +1,28 @@
 title: "Findings"
 type: logs
 streams:
-  - input: cloudbeat
+  - input: cloudbeat/vanilla
```
Comment (ofiriro3) [inline packages/cloud_security_posture/data_stream/findings/manifest.yml:4]: Need to verify before merging it, that the integration is part of the ELK release candidate

Comment (ofiriro3) [reply packages/cloud_security_posture/data_stream/findings/manifest.yml:4]: It is not part of the official release but we set the integration to work only with 8.4 version so no worries regrading it

Comment (kfirpeled) [reply packages/cloud_security_posture/data_stream/findings/manifest.yml:4]: Attached here is the backward compatibility matrix as a reminder, indeed that makes that incompatible with 8.3.0 agents.

As you mentioned, this version is available from `kibana.version` 8.4.0 only, so no issue here.

Screen Shot 2022-01-24 at 11 48 25 (1)

**Review by kfirpeled — COMMENTED** [review]
LGTM,
do not merge before https://github.com/elastic/elastic-agent/pull/638

**Review by kfirpeled — APPROVED** [review]

**Pull request merged**